lib/zone-spec/long-stack-trace.ts

```diff
@@ -136,6 +136,151 @@ function captureStackTraces(stackTraces: string[][], count: number): void {
   }
 }
 
+const zoneAwareStackFrames = {};
```

Comment:
I think we should just roll this into the existing stack frame filtering in the `zone.ts` I don't think that having it in `long-stack-trace` is the right place. If it is in the `zone.ts` than `long-stack-trace` will get it automatically.